clients/src/main/java/org/apache/kafka/common/security/kerberos/KerberosLogin.java

```diff
@@ -82,9 +82,25 @@
 
     public void configure(Map<String, ?> configs, JaasContext jaasContext) {
         super.configure(configs, jaasContext);
-        this.ticketRenewWindowFactor = (Double) configs.get(SaslConfigs.SASL_KERBEROS_TICKET_RENEW_WINDOW_FACTOR);
-        this.ticketRenewJitter = (Double) configs.get(SaslConfigs.SASL_KERBEROS_TICKET_RENEW_JITTER);
-        this.minTimeBeforeRelogin = (Long) configs.get(SaslConfigs.SASL_KERBEROS_MIN_TIME_BEFORE_RELOGIN);
+
```

Comment:
Are we sure we want to make them required config for `KerberosLogin`? I noticed that we have actually defined default value for these configs in `SaslConfigs.addClientSaslSupport()`. These default values are used by producer and consumer.